Structure_Engine/Query/DescriptionOrName.cs

```diff
@@ -42,6 +43,29 @@ public static partial class Query
         /**** Public Methods                            ****/
         /***************************************************/
 
+        [Description("Gets the name from a section. If null or empty, a default description name is provided instead.")]
+        [Input("section", "The section go get the name or default description from.")]
```

Comment:
"The section to get the name or default description from."
"The section whose name we desire" - OK that's a little personal for a software help message